apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+import DesignTokenCard from "./DesignTokenCard.vue";
+
+const tokens = [
+  "onyx-radius-xs",
+  "onyx-radius-sm",
+  "onyx-radius-md",
+  "onyx-radius-lg",
+  "onyx-radius-xl",
+  "onyx-radius-full",
+];
+</script>
+
+<template>
+  <div class="tokens">
+    <DesignTokenCard v-for="borderRadius in tokens" :key="borderRadius" :name="borderRadius">
+      <figure
+        class="radius"
+        :class="{ 'radius--large': borderRadius.endsWith('-xl') || borderRadius.endsWith('-full') }"
+        :style="{
+          borderTopRightRadius: `var(--${borderRadius})`,
+          borderBottomRightRadius: `var(--${borderRadius})`,
+        }"
+      ></figure>
+    </DesignTokenCard>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.tokens {
+  display: flex;
+  flex-direction: column;
+  gap: var(--onyx-spacing-md);
+}
+
+.radius {
+  width: 4rem;
+  height: 4rem;
+  border-width: 1px 1px 1px 0;
+  border-color: var(--onyx-color-base-warning-500);
+  border-style: solid;
+  background: var(--onyx-color-base-neutral-100);
```

Comment:
in darkmode, the background color is not really visible :O

<img width="98" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/70a54b65-9f62-462f-b90b-261ef35cb28e">

maybe an adjustment in the design tokens is needed?

Reply:
#199